pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/ScalarTransformFunctionWrapperTest.java

```diff
@@ -262,6 +262,27 @@ public void testIsNullOperator() {
     testTransformFunction(transformFunction, expectedValues);
   }
 
+  @Test
+  public void testCaseWhenTransformFunctionNullColumn() {
```

Comment:
Added the test for `casewhen` null column, and it's working fine. However, I am not sure
why do we have `casewhen` in ObjectFunctions when we already have a separate implementation? In what scenario will it get used?